src/traces/scattermapbox/attributes.js

```diff
@@ -18,6 +19,48 @@ module.exports = overrideAll({
     lon: scatterGeoAttrs.lon,
     lat: scatterGeoAttrs.lat,
 
+    cluster: {
+        enabled: {
+            valType: 'boolean',
+            dflt: false,
```

Comment:
Should we use a smart default here? ie `true` if any other `cluster` attributes are set, otherwise `false`?

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/5827/commits/477d2a088c44aaa80cfe6ecab64d74df09e342ae.